cli/command/system/version.go

```diff
@@ -32,6 +32,7 @@ Client:{{if ne .Platform.Name ""}} {{.Platform.Name}}{{end}}
  Git commit:	{{.GitCommit}}
  Built:	{{.BuildTime}}
  OS/Arch:	{{.Os}}/{{.Arch}}
+ Context:	{{.Context}}
```

Comment:
I wonder if we really need this `context` in `version` 🤔 
I think that version should mostly print static informations, but `context` is by nature purely dynamic.
That said, it may help the user to understand which daemon she's talking to, so which daemon version are printed.
I'm a bit over the fence 🤷

Reply:
My main reason for putting it here, is that `docker info` requires a daemon connection (and is rather heavy-weight), so if someone wants to use it to set their prompt (as the example), they likely wouldn't want it to be taken from `docker info`.

We _could_ add, e.g., the option to call `docker context use` (without argument) to print the current context, or add a `docker context show-current` subcommand (but not sure if the last one is "great" to have a subcommand _just_ for that); WDYT?